cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,6 +15,21 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+CHECK_LOGIN_NODES_SCRIPT_PATH="<%= node['cluster']['scripts_dir'] %>/slurm/check_login_nodes_status.sh"
+
+# Create a temporary file to store the output
+TEMP_FILE=$(mktemp)
+
+# Check if login nodes are running
+$CHECK_LOGIN_NODES_SCRIPT_PATH > $TEMP_FILE
```

Comment:
Why aren't you just checking the return code of the bash script? E.g.: https://stackoverflow.com/questions/49849957/bash-conditional-based-on-exit-code-of-command

Reply:
Thanks Jacopo, I refactor the logic here and the checking LoginNodes script. If exit 0, then means LoginNodes are stopped, so that we can continue the rotation script.